File: Swift/AoC2024/AoC2024/Solutions/Day00.swift
//
//  Day00.swift
//  AoC 2023
//
//  Created by Simon Biickert on 2023-04-26.
//

import Foundation

class Day00: AoCSolution {
    override init() {
        super.init()
        day = 0
        self.name = "Test Solution"
        self.emptyLinesIndicateMultipleInputs = true
    }
	
	override func solve(_ input: AoCInput) -> AoCResult {
		super.solve(input)
		return AoCResult(part1: "hello", part2: "sync")
	}
}

File: Scala/AoC2024/src/main/scala/Day00.scala
import AoCLib.AoCUtil

class Day00(day: Int, name: String) extends AoCLib.Solution(day, name):
  override def solve(test: Boolean, index: Int): Unit =
    val input = AoCUtil.readGroupedInput(AoCUtil.fileName(day, test))
    println(input)


    
